x-pack/test/cloud_security_posture_functional/agentless/mock_agentless_api.ts

```diff
@@ -10,7 +10,7 @@ import { createServer } from '@mswjs/http-middleware';
 import { http, HttpResponse, StrictResponse } from 'msw';
 
 export const setupMockServer = () => {
-  const server = createServer(deploymentHandler);
+  const server = createServer(undefined, deploymentHandler);
```

Comment:
there is a breaking change in `10.0.1 ->10.0.2` which should be fixed in https://github.com/mswjs/http-middleware/pull/51/files, not sure if we should make this change and then revert, or ignore updates from `mswjs/http-middleware` for the time being